Bottom: 833ddabb12146cf8d987cebc154ab4699f783a21
Top:    cd3bc7da493e1d5d7163e2b22ef19bc7b9e51325
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-02-28 16:07:06 +0800

Refresh of refresh-temp-0

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 1cb8057..9bbf35f 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -649,11 +649,11 @@ static struct sk_buff *receive_mergeable(struct net_device *dev,
 		u32 act;
 
 		/* This happens when rx buffer size is underestimated
-		 * or headroom is not enough because the buffer
+		 * or headroom is not enough because of the buffer
 		 * was refilled before XDP is set. In both cases,
 		 * for simplicity, we will offload them to generic
-		 * XDP routine. This should only happen for several
-		 * packets, so we don't care much about its
+		 * XDP routine. This should only happen for the first
+		 * several packets, so we don't care much about its
 		 * performance.
 		 */
 		if (unlikely(num_buf > 1 ||
